README.md

```diff
@@ -42,6 +42,12 @@ in case arp tables are overflowing:
 sysctl net.ipv4.neigh.default.gc_thresh3=32000
 ```
 
+Compose V2 users should spin up the containers with the following command:
+
+```
+docker-compose --compatibility up -d
```

Comment:
mind elaborating. is this because some of the things we are using are not compatible? any idea which ones?

Reply:
Yes! It's the issue I had with having "-" instead of "_" in the container names as word separator: https://docs.docker.com/compose/migrate/

Apparently in Compose 2 they changed the "_" for a "-" as word separator. With the `compatibility` flag, it returns to the original behavior. If containers do not match in the different platforms then neither the traffic script nor Prometheus would find the containers.